bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -49,30 +49,66 @@
+                {% endif %}
+              </p>
             </div>{#-- /.version-message-container --#}
           </header>
           <div class="main-content">
             <h1 class="hide-for-refresh">{{ _('Browse Freely') }}</h1>
+            <section id="download-buttons">
+              {# handles desktop users (windows, os x, linux) #}
+              <div class="download-button-wrapper desktop" id="download-button-wrapper-desktop">
+                {{ download_firefox(alt_copy=_('Free Download')) }}
+              </div>
+
+              {# handles Android and iOS users #}
+              <div class="mobile download-button-wrapper" id="download-button-wrapper-mobile" data-upgrade-subtitle="{{_('Get it free on Google Play')}}">
+                {{ download_firefox(dom_id='download-button-mobile') }}
+              </div>
+            </section>{#-- /#download-buttons --#}
             <div id="refresh-firefox-wrapper">
               <h2>{{ _('Give Firefox a tune up') }}</h2>
               <button type="button" id="refresh-firefox" data-interaction="Refresh Firefox" data-element-action="Firefox Desktop" data-button-name="Refresh Firefox" data-cta-position="Primary" class="button">{{ _('Refresh Firefox') }}</button>
               <a href="https://support.mozilla.org/kb/refresh-firefox-reset-add-ons-and-settings?utm_source=mozilla.org&amp;utm_medium=referral&amp;utm_campaign=learn-more-link" rel="external">{{ _('Learn more') }}</a>
             </div>
             <div class="desktop-latest-links-wrapper latest-links-wrapper">
               <ul>
-                <li class="hide-for-refresh"><a class="more" href="{{ url('firefox.desktop.index') }}">{{_('Looking for more info?')}}</a></li>
                 <li><a class="more" href="https://support.mozilla.org/products/firefox?utm_source=mozilla.org&amp;utm_medium=referral&amp;utm_campaign=need-help-link">{{_('Need help?')}}</a></li>
                 <li><a class="more" href="{{ firefox_url('desktop', 'all') }}#{{ LANG }}">{{_('Want a fresh copy?')}}</a></li>
               </ul>
-              <div class="hide-for-refresh">
-                {% include 'firefox/new/includes/mobile-promo.html' %}
-              </div>
             </div>
             <div class="android-latest-links-wrapper latest-links-wrapper">
               <ul>
                 <li><a class="more" href="{{ url('firefox.android.index') }}">{{_('Learn more about Firefox for Mobile')}}</a></li>
               </ul>
             </div>
+            <div class="desktop-pre-release-links-wrapper latest-links-wrapper">
+              <ul>
+                <li><a class="more" href="https://support.mozilla.org/products/firefox/install-and-update-firefox">
+                  {% if l10n_has_tag('firefox_new_pre_release') %}
+                    {{_('Update Firefox to the latest version')}}
+                  {% else %}
+                    {{_('Update your Firefox')}}
```

Comment:
@peiying2 - I'm assuming this is ok to use as a fallback string? I got it from [download_button.lang](https://transvision.mozfr.org/?sourcelocale=en-US&locale=en-US&repo=mozilla_org&search_type=entities&recherche=mozilla_org/download_button.lang:5f8fd432&perfect_match=perfect_match)